main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -96,17 +96,35 @@ DataTableCellUI.prototype._render = function() {
       nonstringSpan.className = 'data-table-value-nonstring';
       nonstringSpan.textContent = cell.v;
       divContent.appendChild(nonstringSpan);
-    } else if (URL.looksLikeUrl(cell.v)) {
-      var url = document.createElement('a');
-      url.textContent = cell.v;
-      url.setAttribute('href', cell.v);
-      url.setAttribute('target', '_blank');
-      divContent.appendChild(url);
     } else {
-      var span = document.createElement('span');
-      span.textContent = cell.v;
-      divContent.appendChild(span);
-    }
+      var arr = cell.v.split(" ");
```

Comment:
> The cell's value should be shown as text and not as a valid hyperlink.
(I think what is happening is that our current code just looks for a string beginning with http:// or https:// and doesn't care about validation or remaining characters such as whitespace after that, but should)
I think this is mentioned elsewhere in another related issue but couldn't find it.
Browsers now have native support for URL detection and we should take advantage of that.

Can you please see the desired behaviours, if I have the text as **https://www.wikidata.org/wiki/Property:P670 https://www.wikidata.org/wiki/Property:P669 are now mapped to https://schema.org/streetAddress via https://www.wikidata.org/wiki/Property:P2235** it should be displayed as a `span`, not a `hyperlink`? What the logic does over here is first splits the complete text by space and then checks the each as URL function and then appends them to a new array and other non-matching elements to another array bucket which in itself is quite complex logic. Also, the current logic can also be improved further rather than having a for loop to iterate over an array, the changes should be done in looksLikeURL function to not do the judgement on the basis of protocol prefix.
I would still rely on @wetneb and @thadguidry feedback on this behaviour.

Reply:
The behavior should match the ticket, but I think the current implementation is better than what's proposed on the ticket, so I'd suggest that we update the ticket instead of the code. I think the main thrust of the bug report is that multiple URLs or a URL followed by extra text shouldn't be hyperlinked as a single HREF, which this satisfies.

Reply:
"  https://www.wikidata.org/wiki/Property:P670 https://www.wikidata.org/wiki/Property:P669 are now mapped to https://schema.org/streetAddress via https://www.wikidata.org/wiki/Property:P2235 " 
i think the desired behaviour is ,it should be able to uniquely identify each text component and each url ,and it does so .
<img width="500" alt="Screenshot 2021-02-10 at 7 40 42 AM" src="https://user-images.githubusercontent.com/52077474/107454812-b5c61200-6b73-11eb-9710-4a073625d380.png">

also i think if the text contain both url and normal tag , should it entirely be in <a> or <span> tag ,i guess  each component of it should be assigned a tag depending upon what it is ,whether a text or an url .

if i am getting anything wrong please let me know . @kushthedude 
<img width="633" alt="Screenshot 2021-02-10 at 7 48 56 AM" src="https://user-images.githubusercontent.com/52077474/107455262-8b288900-6b74-11eb-8cf4-4f691209ac84.png">

it may seem like its adding extra spaces or manipulating the data but please look to this